src/forms/reduxForm/ReduxTextInput.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import TextInput from '../TextInput';
+
+/**
+ * @param {Object} props - props passed in from parent and redux-form
+ * @description wraps standard TextInput web component for use with redux-form
+ * deconstructs props that redux-forms sets and sets them on TextInput
+ * @return {Component} TextInput
+ */
+const ReduxTextInput = props => {
```

Comment:
Since this isn't a named function, it's probably safer to manually add a `displayName` to this component (may come in handy with debugging later on)

```js
ReduxTextInput.displayName = 'ReduxTextInput';
```